zproject/prod_settings_template.py

```diff
@@ -409,8 +409,9 @@
 #S3_REGION = None
 #S3_ENDPOINT_URL = None
 
-# Maximum allowed size of uploaded files, in megabytes.  DO NOT SET
-# ABOVE 80MB.  The file upload implementation doesn't support chunked
+# Maximum allowed size of uploaded files, in megabytes.The maximum imposed limit
+# on the nginx config is 80mb.  DO NOT SET ABOVE 80MB.  The file
+# upload implementation doesn't support chunked
 # uploads, so browsers will crash if you try uploading larger files.
```

Comment:
We should make sure we keep the spaces after the period, for readability.  But I think we can also drop the dire warning (since nginx prevents anything horrible) and tie the sentences together to better explain the "why" of the limit.  This made me ask how we chose 80MB as the limit, which led me (via `it log -S80 --follow -p zproject/prod_settings_template.py`) back to #276.  Which teaches me that this was a jquery-filedrop limit, and the value of 80 was chosen since to provide some buffer below 100MB, which is when the failures started to show up.  Though that was before we switched to Uppy (#13323), which may have affected this threshold -- though it's still not a chunked upload.

So with that in mind, perhaps:
```suggestion
# Maximum allowed size of uploaded files, in megabytes.  This value is
# capped at 80MB in the nginx configuration, because the file upload
# implementation doesn't use chunked uploads, and browsers may crash
# with larger uploads.
```

Reply:
Thanks @alexmv for the explanation and suggestions. I have committed your changes to the PR.